rollup/main/default/classes/RollupEvaluatorTests.cls

```diff
@@ -262,7 +259,7 @@ private class RollupEvaluatorTests {
     Opportunity oppThree = new Opportunity(ForecastCategoryName = 'Best Case');
     Opportunity oppFour = new Opportunity(ForecastCategoryName = 'Commit');
     Opportunity oppFive = new Opportunity(Amount = 5, ForecastCategoryName = 'Closed');
-    Formula.recalculateFormulas(new List<Opportunity>{ oppOne, oppTwo, oppThree, oppFour, oppFive });
+    System.Formula.recalculateFormulas(new List<Opportunity>{ oppOne, oppTwo, oppThree, oppFour, oppFive });
```

Comment:
Ha - I actually added this because I was (and am) using the `apex-assist` VS Code plugin, and [there was an issue with its linting of this method](https://github.com/nawforce/apex-link/issues/31) which, while closed, has yet to be released to the plugin. I can revert this - good catch!